src/server/ApiRoutes/Search.js

```diff
@@ -34,19 +31,19 @@ function search(searchKeywords, page, sortBy, order, field, filters, cb, errorcb
     page,
   });
 
-  const aggregationQuery = `/aggregations?${apiQuery}`;
+  // const aggregationQuery = `/aggregations?${apiQuery}`;
   const queryString = `?${apiQuery}&per_page=50`;
 
-  axios
-    .all([axiosApiCall(aggregationQuery), axiosApiCall(queryString)])
-    .then(axios.spread((facets, response) => {
-      cb(facets.data, response.data, page);
-    }))
+  // Also need to make an async call to with aggregationQuery eventually...
```

Comment:
This isn't a requirement right now, to get filters, but when we add it back in, we need to update this.